### PR TITLE
Feature/7062 table page ticket forms - Feature/7002 table page ticket round robin

### PR DESCRIPTION
**Pull request by pxzdev**
**Story 7062**

Showing ticket forms aka layouts.

Design: Channels -> Forms
https://www.figma.com/file/kHciuVpmrvJOboGef98en1/Channels-%E2%86%92-Forms?node-id=0%3A1

**Tasks**

- Page Route: /tickets/forms

- New Link: /tickets/forms/new

- Nav: Channels > Ticket Forms

- Table Field: Name

- Table Field: Departments

- Table Field: ID

- Sortable Field: Name (Default)

- Filterable: Name

- Filterable: Departments

- Add validation if required

___
**Story 7002**

Showing round robins for tickets

Design: Business Rules -> Round Robin

https://www.figma.com/file/PsCPf2tehQ1Fj3UwBPIcz0/Business-Rules-%E2%86%92-Round-Robin?node-id=0%3A1

**Tasks**

- Page Route: /tickets/round-robin

- New Link: /tickets/round-robin/new

- Nav: Tickets > Round Robins

- Table Field: Name

- Table Field: Online Agents Only

- Table Field: Agents

- Table Field: ID

- Sortable Field: Name (Default)

- Filterable Field: Name

- Filterable Field: Agents

- Add validation if required

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1 @@
-REACT_APP_API_URL=https://daily-newadmin.deskprodemo.com/admin-api/graphql
```
Comment (richardhealy) [inline .env]: Why did this change? I think Ruslan's code gets into daily build anyway

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -157,51 +157,51 @@ const Header: FC<IProps> = ({
               <FormattedMessage id={description} />
             </HeaderDescription>
           )}
-          {links && links.length && (
```
Comment (richardhealy) [inline src/components/Header/Header.tsx]: Why was this removed? If there are no links, we shouldn't display the HeaderOptions, right?

Comment (pxzdev) [reply src/components/Header/Header.tsx]: > Why was this removed? If there are no links, we shouldn't display the HeaderOptions, right?

Because it's affect to the buttons on right side, the buttons won't be display, Richard.

File: .env (hunk 1 above)
Comment (pxzdev) [reply .env]: > Why did this change? I think Ruslan's code gets into daily build anyway

Yes, I reverted.

**Comment by richardhealy** [comment]
If no groups, we shouldn’t show the Groups button I think.
